content/en/blog/_posts/Monitoring Kubernetes workloads: the sidecar pattern

```diff
@@ -0,0 +1,174 @@
+
+First, deploy your NGINX deployment using `kubectl apply`:
+
+```
+$ kubectl create namespace webinar
+$ kubectl --namespace webinar apply -f kubernetes/nginx-deployment.yaml
+$ kubectl --namespace webinar get services
+NAME    TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+nginx   LoadBalancer   10.27.251.183   35.230.122.31   8000:30161/TCP   33d
+```
+
+The NGINX container by default runs on port 80. I expose that outside of the Kubernetes cluster at port 8000.
+
+Now, visit your example application in your browser, accessible at [http://35.230.122.31:8000](http://35.230.122.31:8000) And, voilà! We’ve launched a new service, all powered by Kubernetes.
+
+![Welcome-to-nginx](https://user-images.githubusercontent.com/38439765/76876437-ee644b00-683f-11ea-90e3-a3c5a344dc58.png)
+
+This is the stage that a lot of you may have reached: You’ve deployed an app or service in Kubernetes, but want to figure out the best way to monitor it. 
+
+Let’s look at how to add the Sensu Go agent as a sidecar to your application. (This works with all Kubernetes controllers: deployments, Statefulsets, DaemonSets, and so on.) 
+
+Update your deployment using `kubectl apply`:
+
+```
+$ kubectl --namespace webinar apply -f 
+kubernetes/nginx-deployment-with-sensu-sidecar.yaml
+```
+
+Visit the Sensu dashboard, and you’ll see Sensu auto-register the sidecars. (Sensu also automatically de-registers nodes, so you can tell the difference between a node that’s down versus one that’s been de-provisioned.)
+
+![Sensu dashboard monitoring Kubernetes](https://user-images.githubusercontent.com/38439765/76876664-3d11e500-6840-11ea-8fac-baa96eb5bbc9.png)
+
+Here’s the excerpt from the `.spec.template.spec.containers` scope in the Kubernetes deployment that makes this possible: 
+
+```
+- name: sensu-agent
+  image: sensu/sensu:5.11.1
+  command: ["/opt/sensu/bin/sensu-agent", "start", "--log-level=debug", "--insecure-skip-tls-verify", "--keepalive-interval=5", "--keepalive-timeout=10"]
+  env:
+  - name: SENSU_BACKEND_URL
+    value: wss://sensu-backend-0.sensu.sensu-system.svc.cluster.local:8081 wss://sensu-backend-1.sensu.sensu-system.svc.cluster.local:8081 wss://sensu-backend-2.sensu.sensu-system.svc.cluster.local:8081
+  - name: SENSU_NAMESPACE
+    value: webinar
+  - name: SENSU_SUBSCRIPTIONS
+    value: linux kubernetes nginx
+  - name: SENSU_DEREGISTER
+    value: "true"
+  - name: SENSU_STATSD_EVENT_HANDLERS
+    value: statsd
+```
+
+Now that you have this configuration in place, you can scale resources easily. 
+
+```
+kubectl --namespace webinar scale deployment nginx --replicas 10
```

Comment:
```suggestion
$ kubectl --namespace webinar scale deployment nginx --replicas 10
```

This is different from the style guide but matches the rest of the article, so I'm adding the `$`.